src/lib/bootloader/exceptions.rb

```diff
@@ -7,4 +9,17 @@ def initialize(bootloader_name)
       @bootloader_name = bootloader_name
     end
   end
+
+  class BrokenConfiguration < RuntimeError
+    include Yast::I18n
+    attr_reader :reason
+
+    def initialize(msg)
+      @reason = msg
+      textdomain "bootloader"
+
+      super _("Found problem during reading bootloader configuration files. " \
+        "Please open bootloader module and fix it. Details: %s") % msg
```

Comment:
That "during reading" sounds weird. What about "Error reading the bootloader configuration files"?

In the second sentence the meaning of "open bootloader module" could be obvious for us, but not so sure for all users. Maybe, "Please use the bootloader YaST module to fix it", or "...the bootloader module of YaST...", or "the YaST bootloader module..." or simply  "Please use YaST2 bootloader to fix it".